dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/impl/NettyFrameworkImpl.java

```diff
@@ -161,7 +164,7 @@ public void serverStopping() {
             // set up future to log channel stopped messages
             for (Channel channel : activeChannels) {
                 channel.closeFuture().addListener(future -> {
-                    logChannelStopped(channel);
+                    stop(channel);
```

Comment:
So the netty impl seems to do the actual stop sort of out-of-band/aych via a listener event whereas the CF impl used to do the actual stop more in-band (unless there was nothing to do for it)..interesting...I wonder if the 'order' or 'context' of stopping is ever important as we will have lost that perhaps over the event listener technique.